### ISSUE TITLE
Add Travis CI support

### ISSUE THREAD

**Issue by jefftriplett** [simplworld/simpl.world.website#31]
I added a link + accessibility checker that runs on every PR. It'll be broken for a while but it should be 💚  before we "launch".

**Comment by jefftriplett** [comment]
This ticket is to collected the small fixes in #30

**Comment by jefftriplett** [comment]
🎉  this works now. we just need to fix links to make the errors stop.
